### PR TITLE
Add Install-standalone-lava-instance.md

### PR DESCRIPTION
**Pull request by chase-qi**
Signed-off-by: Chase Qi <chase.qi@linaro.org>

### CODE REVIEW

**Comment by chase-qi** [comment]
@mwasilew Here is the step-by-step installation guide for a standalone LAVA instance. Adding generic devices qemu and lxc included.  Would you please take a look? Please let me know if anything wrong or missing.

File: LAVA/Install-standalone-lava-instance.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    apt install -y postgresql
+    apt install -y -t stretch-backports lava-server
+    ```   
+    
+6. Add local super user.
+
+    ```bash
+    root@stretch-lava:~# lava-server manage createsuperuser
+    Username (leave blank to use 'root'): admin
+    Email address: chase.qi@linaro.org
```
Comment (mwasilew) [inline LAVA/Install-standalone-lava-instance.md]: maybe use john.doe@example.com

File: LAVA/Install-standalone-lava-instance.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    sed -i '/{/a\    "CSRF_COOKIE_SECURE": false,\n    "SESSION_COOKIE_SECURE": false,' /etc/lava-server/settings.conf
+    ```
+    
+    Restart `lava-server-gunicorn` service to put the changes into effects.
+    
+    ```bash
+    systemctl restart lava-server-gunicorn
+    ```
+    
+    Open link http://localhost/, you should be able to sing in with the super user created in the previous step.
```
Comment (mwasilew) [inline LAVA/Install-standalone-lava-instance.md]: s/sing/sign/

File: LAVA/Install-standalone-lava-instance.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    Worker Host: stretch-lava
+    Device Version: v1.0
+    User: admin
+    User with physical access: admin
+    Select 'Is public'
+    Select 'Pipeline device'
+    Health: Unknown
+    ```
+
+3. Add device `CONFIG` file(aka device dictionary).
```
Comment (mwasilew) [inline LAVA/Install-standalone-lava-instance.md]: space missing before opening bracket

**Review by mwasilew — CHANGES_REQUESTED** [review]
It might be a good idea to make the lines a bit shorter. But apart from that and a few typing mistakes it looks good.

**Comment by chase-qi** [comment]
> It might be a good idea to make the lines a bit shorter. But apart from that and a few typing mistakes it looks good.

I have fixed your comments and shorten the content line length < 80 and code&&link line length <120.

Thanks,
Chase

**Comment by mwasilew** [comment]
Looks good. However I don't have enough rights to merge this patch.

**Comment by chase-qi** [comment]
I though you have the permissions as you pushed some commits before. You are admin now :)

**Review by mwasilew — APPROVED** [review]

**Pull request merged**